docs/reference/config-params.md

```diff
@@ -280,462 +324,543 @@ Otherwise it will default to `1`.
+      workload:
+        livenessProbe:
+          type: http
+          http:
+            port: 8080
 ...
 ```
 
-## kev.workload.liveness-probe-http-path
+### x-k8s.workload.livenessProbe.http.path
 
 Defines the liveness probe path to be used for the workload when the type is `http`. 
 See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request). 
 
-### Possible options: String
+#### Possible options: String
 
-> kev.workload.liveness-probe-http-path:
+> x-k8s.workload.livenessProbe.http.path:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-http-path: /status
+    x-k8s:
+      workload:
+        livenessProbe:
+          type: http
+          http:
+            port: 8080
+            path: /status
 ...
 ```
 
-## kev.workload.liveness-probe-tcp-port
+### x-k8s.workload.livenessProbe.tcp.port
 
-Defines the liveness probe path to be used for the workload when the type is `tcp`. 
-See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request). 
+Defines the liveness probe port to be used for the workload when the type is `tcp`.
+See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-tcp-liveness-probe).
 
-### Possible options: Integer
+#### Possible options: Integer
 
-> kev.workload.liveness-probe-tcp-port:
+> x-k8s.workload.livenessProbe.tcp.port:
 ```yaml
 version: 3.7
 services:
   my-service:
-    labels:
-      kev.workload.liveness-probe-tcp-port: 8080
+    x-k8s:
+      workload:
+        livenessProbe:
+          type: tcp
+          tcp:
+            port: 8080
 ...
 ```
 
-## kev.workload.liveness-probe-interval
-
-Defines how often liveness proble should run for the workload. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the interval from the information specified in the compose file.
+### x-k8s.workload.livenessProbe.failureThreshold
 
-Kev uses the following heuristics to derieve that information for each service:
+Defines the liveness probe's failure threshold (retries) for the workload. See official K8s [documentation](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-command). Kev will attempt to infer the timeout value from the information specified in the compose file.
 
-If compose file(s) specifies the `healthcheck.interval` attribute key in a service config it will use its value.
-Otherwise it'll default to `1m` (1 minute).
-
-### Default: `1m`
+#### Default: `3`
 
-### Possible options: Time duration
+#### Possible options: Arbitrary time duration. Example: `5`
```

Comment:
I guess this was applicable to `interval` setting? Perhaps sections got mixed up a little?

Reply:
`interval` is now `period`.